src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -130,6 +134,25 @@ PortTooltipContent.propTypes = {
   ports: PropTypes.object,
 };
 
+const LogDialogContent = ({ urlLists }) => {
+  const lists = [];
+  for (const p of urlLists) {
+    lists.push(p);
+  }
+  const urlpairs = lists.map(lists => (
+    <dl key='log-list'>
+      <Link href={lists.uri} styles={{ root: [FontClassNames.mediumPlus] }}>
```

Comment:
Can you change to open the link in a new tab? Maybe add `target="_black` can works.